src/Entities/Payments/SofortPayment.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Entities\Payments;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @since 6.0
+ *
+ * @ORM\Table(name="payment_sofort")
```

Comment:
I was thinking about how to namespace the payment class earlier.

    WMDE\Fundraising\DonationContext\SomeDatabaseStuff\SofortPayment